portfolio/src/main/webapp/feedback.html

```diff
@@ -15,14 +15,17 @@
     </div>
     <div id="content">
       <h1>Feedback and Comments</h1>
-      <form action="/data" method="POST">
+      <form id="post-comment-form" action="/data" method="POST">
         <h2>Comment:</h2>
-        <input type="text" name="username" required placeholder="Enter username to be displayed..." maxlength=40>
-        <br>
-        <br>
-        <textarea type="text" name="comment" required style="height:150px" placeholder="Write a comment..."></textarea>
+        <div id="post-comment">
+          <input type="text" name="username" required placeholder="Enter username to be displayed..." maxlength=40>
```

Comment:
Would be cool if the API call can return a username (or the email) and you auto populate here to prevent users from posting comments using someone else's name.